packages/maskbook/src/plugins/FileService/types.ts

```diff
@@ -1,5 +1,6 @@
 export interface FileInfo {
-    type: 'arweave'
+    provider: 'arweave' | 'skynet'
+    type: 'file'
```

Comment:
hmmm although this is a good design, it will break the file plugin cause the `type` field is used as a storage key. cc @septs maybe we should add a migration for this (and land it in the master branch so this PR won't to have to consider this)

Reply:
I don't known how migrate plugin database. Can you come?

Reply:
ok I'll handle this tomorrow